guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -11,139 +11,576 @@
     "# Part 1 - Introduction to GeoEnrichment"
```

Comment:
<h3>GeoEnrichment Source </h3><p> </p><p>Utilizing these capabilities requires either a properly configured Web GIS instance and a login with permissions to utilize these capabilities, or a local installation of <a href="https://doc.arcgis.com/en/business-analyst/desktop/introduction-to-esri-business-analyst-desktop-install-guide.htm" target="_blank">ArcGIS Pro with Business Analyst</a> and at least one country <a href="https://doc.arcgis.com/en/business-analyst/desktop/installing-business-analyst-data.htm" target="_blank">data pack installed</a>. A properly configured Web GIS instance can either use ArcGIS Online or ArcGIS Enterprise. ArcGIS Enterprise can support the GeoEnrichment module capabilities by configuring the GeoEnrichment utility service to <a href="https://enterprise.arcgis.com/en/portal/latest/administer/windows/configure-services.htm#ESRI_SECTION2_1E0134BF60A049FFB388265B5A6AAE7F" target="_blank">connect to ArcGIS Online</a> or utilize a fully configured <a href="https://doc.arcgis.com/en/business-analyst/enterprise/latest/windows/business-analyst-enterprise-overview.htm" target="_blank">ArcGIS Business Analyst Enterprise</a> deployment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>